core/templates/domain/question/QuestionObjectFactory.ts

```diff
@@ -194,7 +198,7 @@ export class QuestionObjectFactory {
 
   createDefaultQuestion(skillIds: string[]): Question {
     return new Question(
-      null, this.stateObject.createDefaultState(null),
+      'id', this.stateObject.createDefaultState('state'),
```

Comment:
Ditto, you cannot make up random IDs and state names.

Reply:
> Ditto, you cannot make up random IDs and state names.

Same here we can give empty strings as parameters.